imap_processing/swe/l2/swe_l2.py

```diff
@@ -330,42 +330,48 @@ def swe_l2(l1b_dataset: xr.Dataset, data_version: str) -> xr.Dataset:
         * swe_constants.ENERGY_CONVERSION_FACTOR,
         name="energy",
         dims=["energy"],
-        attrs=cdf_attributes.get_variable_attributes("energy"),
+        attrs=cdf_attributes.get_variable_attributes("energy", check_schema=False),
     )
 
     energy_label = xr.DataArray(
         np.array(list(swe_constants.ESA_VOLTAGE_ROW_INDEX_DICT.keys())).astype(str),
         name="energy_label",
         dims=["energy"],
-        attrs=cdf_attributes.get_variable_attributes("energy_label"),
+        attrs=cdf_attributes.get_variable_attributes(
+            "energy_label", check_schema=False
+        ),
     )
 
     # Angle of each CEM detectors.
     inst_el_xr = xr.DataArray(
         swe_constants.CEM_DETECTORS_ANGLE,
         name="inst_el",
         dims=["inst_el"],
-        attrs=cdf_attributes.get_variable_attributes("inst_el"),
+        attrs=cdf_attributes.get_variable_attributes("inst_el", check_schema=False),
     )
     inst_el_label = xr.DataArray(
         swe_constants.CEM_DETECTORS_ANGLE.astype(str),
         name="inst_el_label",
         dims=["inst_el"],
-        attrs=cdf_attributes.get_variable_attributes("inst_el_label"),
+        attrs=cdf_attributes.get_variable_attributes(
+            "inst_el_label", check_schema=False
+        ),
     )
 
     # Spin Angle bins storing bin center values.
     inst_az_xr = xr.DataArray(
-        np.arange(0, 360, 12) + 6,
+        np.arange(0.0, 359.99, 12) + 6,
```

Comment:
Is this to try and get the right dtype? If so, maybe putting `np.arange(0, 360, 12, dtype=float) + 6` would be more explicit.